src/components/OverviewMenu/OverviewMenu.tsx

```diff
@@ -0,0 +1,217 @@
+import { useEffect, useState } from 'react';
+
+import sortBy from 'lodash/sortBy';
+import { useQuery } from 'react-query';
+import { useNavigate } from 'react-router';
+import { useLocation, NavLink } from 'react-router-dom';
+import { useRecoilValueLoadable } from 'recoil';
+
+import { Popover, makeStyles, Hidden } from '@material-ui/core';
+
+import { linkStyle, menuGroup } from 'components/MainLayout/MainHeader';
+import { scrollToTop } from 'components/MainLayout/MainLayout';
+import isFeatureEnabled from 'config/features';
+import { useApiBase } from 'hooks';
+import { useCurrentOrgId } from 'hooks/gql/useCurrentOrg';
+import useConnectedAddress from 'hooks/useConnectedAddress';
+import { ChevronUp, ChevronDown } from 'icons';
+import { rSelectedCircle } from 'recoilState/app';
+import { useHasCircles } from 'recoilState/db';
+import { paths, isCircleSpecificPath } from 'routes/paths';
+import { Box, Link, Text } from 'ui';
+
+import { getOrgData } from './getOrgData';
+
+import type { Awaited } from 'types/shim';
+
+type QueryResult = Awaited<ReturnType<typeof getOrgData>>;
+
+const useStyles = makeStyles(theme => ({
+  popover: {
+    marginTop: theme.spacing(0.5),
+    padding: 0,
+    borderRadius: 8,
+    background: '#FFF',
+    boxShadow:
+      '0px 0px 3px 0px #0000001C, 0px 0px 16px 0px #0000001F, 0px 0px 87px 0px #0000003D',
+    display: 'flex',
+    flexDirection: 'column',
+    top: '13.5px !important',
+    left: '45px !important',
+    transition: 'none !important',
+  },
+}));
```

Comment:
Can we do this using stitches instead of Material UI?

Reply:
I'll look into transitioning to a radix / stitches popover component